Decode/EventHeaderEnumerator.cs

```diff
@@ -2107,8 +2099,8 @@ private void StartValueStringNul16(ReadOnlySpan<byte> eventDataSpan)
                 }
             }
 
-            m_itemSizeCooked = eventDataSpan.Length;
-            m_itemSizeRaw = eventDataSpan.Length;
+            m_itemSizeCooked = eventDataSpan.Length - m_dataPosRaw;
+            m_itemSizeRaw = eventDataSpan.Length - m_dataPosRaw;
```

Comment:
Note: Minor "bug" fix. Whether this is better or worse is situational/debatable, but the new behavior is the behavior that was originally intended for this situation.
- Old: a nul-terminated string that does not contain nul-termination before the end of the event would be considered an error.
- New: a nul-terminated string that does not contain nul-termination before the end of the event will stop at the end of the event.